main.go

```diff
@@ -359,28 +359,30 @@ func main() {
 	archiveCmd.SetScheme(cfg.Scheme)
 	archiveCmd.SetConfiguration(cfg.Configuration)
 
+	var customOptions []string
 	if cfg.ForceTeamID != "" {
 		log.Printf("Forcing Development Team: %s", cfg.ForceTeamID)
-		archiveCmd.SetForceDevelopmentTeam(cfg.ForceTeamID)
+		customOptions = append(customOptions, fmt.Sprintf("DEVELOPMENT_TEAM=%s", cfg.ForceTeamID))
```

Comment:
These are no longer supported in `go-xcode` because we decided to deprecate these input options in the iOS steps, but I'd rather not remove these inputs from this step and cause a breaking change.